docs/resources/gaussdb_redis_instance.md

```diff
@@ -0,0 +1,179 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud\_gaussdb\_redis\_instance
+
+GaussDB for Redis instance management within HuaweiCoud.
+
+## Example Usage
+
+### create a gaussdb for redis instance with tags
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "instance_1" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+### create a gaussdb redis instance with backup strategy
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "instance_1" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  backup_strategy {
+    start_time = "03:00-04:00"
+    keep_days  = 14
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the Redis instance resource.
+  see [Endpoints](https://developer.huaweicloud.com/endpoint) for more detail.
```

Comment:
please use the intel URL.

Reply:
English page is not found, delete it

Reply:
https://developer.huaweicloud.com/intl/en-us/endpoint?GaussDB%20NoSQL
For all URL, we need to use English page.